doc/admin/deployment.md

```diff
@@ -193,16 +193,98 @@ Refer to the example configuration file ([daos_server.yml](https://github.com/da
 for latest information and examples.
 
 At this point of the process, the servers: and provider: section of the yaml
-file can be left blank and will be populsted in the subsequent sections.
+file can be left blank and will be populated in the subsequent sections.
 
-#### Certificate Generation
+#### Certificate Generation And Configuration
 
 The DAOS security framework relies on certificates to authenticate
-administrators. The security infrastructure is currently under development and
-will be delivered in DAOS v1.0. Initial support for certificates has been added
-to DAOS and can be disabled either via the command line or in the DAOS server
-configuration file. Currently, the easiest way to disable certificate support
-is to pass the -i flag to daos_server.
+components and administrators in addition to encrypting DAOS control plane
+communications. A set of self-signed certificates for a given
+DAOS system may be generated by running the `gen_certificates.sh` script
+provided with the DAOS software if there is not an existing TLS certificate
+infrastructure.
+
+When DAOS is installed from RPMs, this script is provided in the base `daos` RPM, and
+may be invoked in the directory to which the certificates will be written. As part
+of the generation process, a new local Certificate Authority is created to handle
+certificate signing, and three role certificates are created:
+
+```
+# /usr/lib64/daos/certgen/gen_certificates.sh
+Generating Self-Signed Certificate Authority
+Self-Signed CA created in ./daosCA
```

Comment:
Running your own CA is generally not considered self-signed. You are establishing your own root of trust.

Reply:
OK, s/Self Signed/Local/ ?